compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -477,14 +477,33 @@ class ComposeScene internal constructor(
     /**
      * Returns the current content size
      */
+    @Deprecated("Will be removed in 1.7", replaceWith = ReplaceWith("calculateContentSize()"))
```

Comment:
```suggestion
    @Deprecated("Use calculateContentSize() instead", replaceWith = ReplaceWith("calculateContentSize()"))
```
We lean towards to not removing deprecated API until 2.0

Reply:
And better to have `calculateContentSize` experimental for now, with `TODO: makes stable and deprecate contentSize after 1.6`